backend/environment.yml

```diff
@@ -37,5 +37,6 @@ dependencies:
   - git=2.34.1
   - httpx=0.23.0
   - anyio=3.5.0
+  - debugpy=1.8.0
```

Comment:
When you include it here, it can be removed in the entry point script of the background jobs celery worker :)